frontend/src/lib/lemon-ui/LemonCheckbox/LemonCheckbox.scss

```
@@ -31,9 +31,13 @@
             transition: border 200ms ease, background 200ms ease;
             background: var(--bg-light);
             border: 1.5px solid var(--border-bold);
-            border-radius: 3px; // Intentionally a bit smaller than --radius
+            border-radius: 0.1875rem; // Intentionally a bit smaller than --radius
             flex-shrink: 0;
 
+            &.posthog-3000 {
+                border-radius: 0.25rem;
```

Feels like we should go with one  of these two
```suggestion
                border-radius: var(--radius);
```
```suggestion
                border-radius: 0.25rem; // Intentionally a bit smaller than --radius
```

---

Good point